examples/jvm/java/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM maven:3.5-jdk-8 as BUILD
+COPY src /usr/src/myapp/src
+COPY pom.xml /usr/src/myapp
+ENTRYPOINT mvn -f /usr/src/myapp/pom.xml clean package
```

Comment:
You might want to add a simple `build.sh` for building the package to help the users understand what to do. Especially, since you need to do add some non-trivial/easy-to-forget arguments to it (like, volume mounting).

Even better: remove this dockerfile, and simply use the maven image directly in a `build.sh`, like explained here: https://docs.docker.com/samples/library/maven/
Probably something like:

_examples/jvm/java/build.sh_
```bash
docker run -it --rm -v "$(pwd)":/usr/src/myapp -w /usr/src/myapp maven:3.3-jdk-8 mvn clean package
```

Reply:
Oh cool, this is superb. I wanted to do something like this but did not know about this utility! Awesome!!

Reply:
And this is only for test and not intended for users.

Reply:
IMO it is really useful to users to not have to think how to build the example.. it is an example after all. Just add it as a `build.sh` even if you don't use it in anywhere else.

Reply:
Done, thanks